dist/images/ovn-is-leader.sh

```diff
@@ -35,6 +35,22 @@ fi
 # Store ovn-sb leader to svc kube-system/ovn-sb
 if [[ "$ENABLE_SSL" == "false" ]]; then
   sb_leader=$(ovsdb-client query tcp:127.0.0.1:6642 "[\"_Server\",{\"table\":\"Database\",\"where\":[[\"name\",\"==\", \"OVN_Southbound\"]],\"columns\": [\"leader\"],\"op\":\"select\"}]")
+  set +e
+  northd_svc=$(kubectl get svc -n kube-system | grep ovn-northd)
+  if [ -z "$northd_svc" ]; then
+   echo "ovn-northd svc not exist"
+  else
+   northd_leader=$(kubectl get ep -n kube-system ovn-northd -o jsonpath={.subsets\[0\].addresses\[0\].ip})
+   if [ "$northd_leader" == "" ]; then
+      # no available northd leader try to release the lock
+      if [[ "$ENABLE_SSL" == "false" ]]; then
+        ovsdb-client -v -t 1 steal tcp:127.0.0.1:6642  ovn_northd
+      else
```

Comment:
It seems that the location is not right. The `ENABLE_SSL` is false in line 36.

Reply:
move the location